api/pom.xml

```diff
@@ -21,7 +21,7 @@
         <!-- This is just for now and will not work if the API has a separate release cycle than the rest. -->
         <groupId>org.eclipse.microprofile.lra</groupId>
         <artifactId>microprofile-lra-parent</artifactId>
-        <version>1.0-SNAPSHOT</version>
+        <version>0.0.2.Final-SNAPSHOT</version>
```

Comment:
Why do you change the version? The version should stay as 1.0-SNAPSHOT till 1.0 is released.

Reply:
Good point, the change crept in because that is what we are using privately, I will revert this part of the fix, thanks.